packages/pigeon/platform_tests/ios_unit_tests/ios/RunnerTests/AsyncHandlersTest.m

```diff
@@ -8,24 +8,25 @@
 
 ///////////////////////////////////////////////////////////////////////////////////////////
 @interface Value ()
-+ (Value*)fromMap:(NSDictionary*)dict;
-- (NSDictionary*)toMap;
++ (Value *)fromMap:(NSDictionary *)dict;
+- (NSDictionary *)toMap;
```

Comment:
Is there a reason you're changing the style of this file?

Reply:
I just do whatever the CI tells me for this.  Maybe the clang-tidy was upgraded on the bot?

Reply:
Lookingat this again, this looks unnecessary.  Let me try to remove it from the patch.

Reply:
removed